pkg/controller/controller_instance.go

```diff
@@ -220,6 +221,17 @@ func (c *controller) reconcileInstance(instance *v1alpha1.Instance) error {
 				successProvisionMessage,
 			)
 			c.recorder.Eventf(instance, api.EventTypeNormal, successProvisionReason, successProvisionMessage)
+		} else {
+			// the broker returned a failure response
+			errorProvisionCalledMessage := fmt.Sprintf("provision call failed")
+			c.updateInstanceCondition(
```

Comment:
@derekwaynecarr good catch. I'll log and return on these changed portions, and I've created https://github.com/kubernetes-incubator/service-catalog/issues/925 as a follow-up